<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelHalley">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Método de Halley</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFuncaoHalley">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>f(x) = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFuncaoHalley">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDerivadaHalley">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>f'(x) = </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDerivada2Halley">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>f''(x) = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDerivadaHalley">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDerivada2Halley">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalcularHalley">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Xo = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditX0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultadosHalley">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resultados</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRaizHalley">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Raiz: X = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRaizHalley">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gráfico</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidgetHalley" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFuncaoRidders">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>f(x) = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRaizRidders">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gráfico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalcularRidders">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidgetRidders" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFuncaoRidders">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRidders">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Método de Ridders</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRaizRidders">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Raiz: X = </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultadosRidders">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resultados</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX1Ridders">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X1 = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditX1Ridders">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelX2Ridders">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X2 = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditX2Ridders">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
